### ISSUE TITLE
Get values of trained parameters in evaluation

### ISSUE THREAD

**Issue by chenchen718** [ppwwyyxx/tensorpack#517]
How do I get the values of well trained parameters, e.g., the batch normalization mean and variance in the first layer, in evaluation? Which variable are these parameters stored in? I digged into the variables that restore the trained model, but got lost before locating where the parameter values are.

Thanks a lot!

**Comment by ppwwyyxx** [comment]
http://tensorpack.readthedocs.io/en/latest/tutorial/save-load.html#work-with-tf-checkpoint
If you could print the names it should be easy to know which name you need.

**Comment by chenchen718** [comment]
I know the names of the variables I need, my difficulty is how to retrieve the variable values using these names. `tf.train.NewCheckpointReader(path).get_variable_to_shape_map()` seems to only provide a dictionary of the variable name and shape.  Is there a dictionary that can tell me the value corresponding to each name?

**Comment by chenchen718** [comment]
I've found it. I am using the function`dump_chkpt_vars()` to retrieve the values .

Thanks a lot!

**Comment by ppwwyyxx** [comment]
You can also use `NewCheckpointReader.get_tensor(name)`.
